src/reference/asciidoc/mail.adoc

```diff
@@ -55,6 +55,53 @@ Spring Integration provides the `ImapIdleChannelAdapter` which is itself a Messa
 It delegates to an instance of the `ImapMailReceiver` but enables asynchronous reception of Mail Messages.
 There are examples in the next section of configuring both types of inbound Channel Adapter with Spring Integration's namespace support in the 'mail' schema.
 
+[[mail-mapping]]
+=== Inbound Mail Message Mapping
+
+By default, the payload of messages produced by the inbound adapters is the raw `MimeMessage`; you can interrogate
+the headers and content using that object.
+Starting with _version 4.3_, you can provide a `HeaderMapper<MimeMessage>` to map the headers to `MessageHeaders`; for
+convenience, a `DefaultMailHeaderMapper` is provided for this purpose.
+This maps the following headers:
+
+- `mail_from` - A String representation of the `from` address.
+- `mail_bcc` - A String array containing the `bcc` addresses.
+- `mail_cc` - A String array containing the `cc` addresses.
+- `mail_to` - A String array containing the `to` addresses.
+- `mail_replyTo` - A String representation of the `replyTo` address.
+- `mail_subject` - The mail subject.
+- `mail_lineCount` - A line count (if available).
+- `mail_receivedDate` - The received date (if available).
+- `mail_size` - The mail size (if available).
+- `mail_expunged` - A boolen indicating if the message is expunged.
+- `mail_raw` - A `MultiValueMap` containing all the mail headers and their values.
+- `mail_contentTYpe` - The content type of the original mail message.
```

Comment:
And yes: I was right above. We don't have such a mapping in the `DefaultMailHeaderMapper`